sirepo/pkcli/job_supervisor.py

```diff
@@ -96,7 +96,8 @@ def on_close(self):
             pkdlog("error={} {}", e, pkdexc())
 
     async def on_message(self, msg):
-        await _incoming(msg, self)
+        # WebSocketHandler only allows one on_message at a time.
+        asyncio.ensure_future(_incoming(msg, self))
```

Comment:
Elsewhere I think we use create_task which I believe is "preferred" https://github.com/python/asyncio/issues/477#issuecomment-268709555. Maybe we abstract in pkasyncio so we always use the same thing? 

Can the async be removed from the def of on_message? Functions with no awaits but marked async look strange to me...

Reply:
650825be5. Funny. I read [this](https://github.com/xxleyi/learning_list/issues/120) incorrectly.